src/main/java/svc/data/citations/CitationDAO.java

```diff
@@ -1,128 +1,50 @@
-			Citation citation = jdbcTemplate.queryForObject(sql, parameterMap, new CitationSQLMapper());
-			return citation;
-		} catch (Exception e) {
-			LogSystem.LogDBException(e);
-			return null;
-		}
-	}
-	
-	public Citation getByCitationNumberAndDOB(String citationNumber, LocalDate dob) {
-		try {
-			Map<String, Object> parameterMap = new HashMap<String, Object>();
-			parameterMap.put("citationNumber", citationNumber);
-			parameterMap.put("dob", Date.valueOf(dob));
-			String sql = "SELECT * FROM citations WHERE citation_number = :citationNumber AND date_of_birth = :dob";
-			Citation citation = jdbcTemplate.queryForObject(sql, parameterMap, new CitationSQLMapper());
-			
-			return citation;
-		} catch (Exception e) {
-			LogSystem.LogDBException(e);
-			return null;
-		}
-	}
-	
-	public List<Citation> getByDOBAndLicense(LocalDate dob, String driversLicenseNumber) {
-		try {
-			Map<String, Object> parameterMap = new HashMap<String, Object>();
-			parameterMap.put("driversLicenseNumber", driversLicenseNumber);
-			parameterMap.put("dob", Date.valueOf(dob));
-			String sql = "SELECT * FROM citations WHERE date_of_birth = :dob AND drivers_license_number = :driversLicenseNumber";
-			List<Citation> citations = jdbcTemplate.query(sql, parameterMap, new CitationSQLMapper());
-			
-			return citations;
-		} catch (Exception e) {
-			LogSystem.LogDBException(e);
-			return null;
+        List<Observable<Citation>> citationSearches = Lists.newArrayList();
+		for(CitationDataSource source : sources) {
+            citationSearches.add(Observable.from(source.getByCitationNumberAndDOB(citationNumber, dob)).onExceptionResumeNext(Observable.just(null)));
 		}
+
+		return Observable.merge(citationSearches).onExceptionResumeNext(Observable.just(null)).toList().toBlocking().first();
 	}
-	
-	public List<Citation> getByDOBAndNameAndMunicipalities(LocalDate dob, String lastName, List<Long> municipalities) {
-		try {
-			Map<String, Object> parameterMap = new HashMap<>();
-			parameterMap.put("lastName", lastName.toLowerCase());
-			parameterMap.put("dob", Date.valueOf(dob));
-			parameterMap.put("municipalities", municipalities);
 
-			List<Citation> citations = jdbcTemplate.query(getSql("citation/get-by-location.sql"), parameterMap, new CitationSQLMapper());
-			
-			return citations;
-		} catch (Exception e) {
-			LogSystem.LogDBException(e);
-			return null;
-		}
+	public List<Citation> getByLicenseAndDOB(String driversLicenseNumber, LocalDate dob) {
+        List<CitationDataSource> sources = citationDataSourceFactory.getAllCitationDataSources();
+
+        List<Observable<Citation>> citationSearches = Lists.newArrayList();
+        for(CitationDataSource source : sources) {
+            citationSearches.add(Observable.from(source.getByLicenseAndDOB(driversLicenseNumber, dob)));
+        }
+
+        return Observable.merge(citationSearches).onExceptionResumeNext(Observable.just(null)).toList().toBlocking().first();
```

Comment:
It's certainly possible. The data was garbled but I have no guarantee that something could match...but you are correct, the yml property for stlcourts.citationDataSources.testEnabled is hard-coded to false for the production environment. The only environment in which it is dynamic is the test(qa) environment so we can toggle it quickly should they want to demo using some stable data.